packages/homepage/content/docs/3-configuration.md

```diff
@@ -6,7 +6,7 @@ description: "You can configure sandboxes and other options with configuration f
 
 ## Configuration Files
 
-We think that configuration is used when a program is too stupid to figure out what the user really wants, and try to steer away from it as much as possible. However, there are always advanced use cases where the user wants to control every aspect of their program, and thus configure everything. CodeSandbox is a clear example of a program that requires configuration, it doesn't fit the 95%.
+There are always advanced use cases where the user wants to control every aspect of their program, and thus configure everything. CodeSandbox is a clear example of a program that requires configuration, it doesn't fit the 95%.
```

Comment:
Hmm, I think this line is actually a valid line. It tells something about the program, not a person. I have it from here:

https://fishshell.com/docs/current/design.html

Under "Configurability is the root of all evil". We can rewrite it to remove the word, but I think that this sentence is the most important sentence of that paragraph.

Reply:
Changed to: We think that configuration is used when a program can't understand what the user really wants,

Is that okay?

Reply:
Because even if you got it from somewhere it's still a thing where without meaning you are calling all software that requires config stupid

Reply:
> "We think that configuration is used when a program can't understand what the user really wants"

Yes, that's okay! I understand that the `stupid` can offend people, good suggestion to change it to this.

Reply:
It's what happens when you live in berlin for long 😂